webui/react/src/services/apiConfig.ts

```diff
@@ -288,10 +288,12 @@ export const listRoles: DetApi<Service.ListRolesParams, Api.V1ListRolesResponse,
   {
```

Comment:
Is it possible to just check `feature.isOn('rbac')` where we call `listRoles`?

Reply:
I haven't tried out the feature flags yet. Agree our future setup should do something like this in the future.